packages/bpk-component-list/src/BpkList.js

```diff
@@ -16,23 +16,29 @@
  * limitations under the License.
  */
 
+/* @flow strict */
+
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { type Node } from 'react';
 import { cssModules } from 'bpk-react-utils';
 
 import STYLES from './BpkList.scss';
 
 const getClassName = cssModules(STYLES);
 
-const BpkList = props => {
-  const TagName = props.ordered ? 'ol' : 'ul';
-  const classNames = [getClassName('bpk-list')];
+type Props = {
+  children: Node,
+  ordered: boolean,
+  className: ?string,
+};
+
+const BpkList = (props: Props) => {
+  const { children, ordered, className } = props;
 
-  if (props.className) {
-    classNames.push(props.className);
-  }
+  const TagName: any = ordered ? 'ol' : 'ul';
+  const classNames: string = getClassName('bpk-list', className);
```

Comment:
I don't know why but it makes me really happy when we migrate components to use the shorter `getClassName` syntax